### PR TITLE
Fix getNearbyEntities on null. Add default level to Exposition.

### PR DESCRIPTION
**Pull request by xISRAPILx**
### Purpose and Effect of This Pull Request
Fix getNearbyEntities on null. Add default level to Exposition.

### Testing
TESTED!

### CODE REVIEW

**Comment by Rateek** [comment]
+1

**Comment by Leo3418** [comment]
Is this really useful? When will `$center->getLevel()` has value `null`?

**Comment by xISRAPILx** [comment]
Yes. @Leo3418

**Comment by Leo3418** [comment]
When?

**Comment by xISRAPILx** [comment]
@Leo3418 When a person does not show the world in Position, it will put itself.

**Comment by xISRAPILx** [comment]
@NycuRO =)

**Comment by xISRAPILx** [comment]
push? @DerddyDert

**Comment by DerddyDert** [comment]
@xISRAPILx Id like to see @Leo3418 follow up one more time.

**Comment by xISRAPILx** [comment]
Thanks for the review. @SuperXingKong

**Comment by DerddyDert** [comment]
@xISRAPILx can you remove default level code?

**Comment by xISRAPILx** [comment]
This is necessary when the world is not specified, it is in the download or when it is damaged. @DerddyDert

File: src/pocketmine/level/Explosion.php [hunk 1]
```diff
@@ -65,7 +66,7 @@ class Explosion {
 	 * @param bool     $dropItem
 	 */
 	public function __construct(Position $center, $size, $what = null, bool $dropItem = true){
-		$this->level = $center->getLevel();
+		$this->level = $center->getLevel() === null ? Server::getInstance()->getDefaultLevel() : $center->getLevel();
```
Comment (CortexPE) [inline src/pocketmine/level/Explosion.php]: Better way to do this:
```
$this->level = $center->getLevel() ?? Server::getInstance()->getDefaultLevel();
```

**Comment by ivan691** [comment]
works ✓

**Pull request closed without merging**